src/main/java/io/github/syst3ms/skriptparser/registration/SkriptRegistration.java

```diff
@@ -737,6 +692,23 @@ public void register() {
             contextValues.add(new ContextValue<>(context, type, isSingle, pattern, (Function<TriggerContext, R[]>) contextFunction, time, name.startsWith("*")));
             return this;
         }
+
+        /**
+         * Adds this event to the list of currently registered syntaxes
+         */
+        @Override
+        public void register() {
+            for (int i = 0; i < super.patterns.size(); i++) {
+                var pattern = super.patterns.get(i);
+                if (pattern.startsWith("*")) {
```

Comment:
Why not using removePrefix method ?

Reply:
Not really necessary here as it does the same thing. That method is unnecessary anyways.